models/book.rb

```diff
@@ -0,0 +1,11 @@
+class Book
+  # Getter and Setters
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] A method for adding rentals should be present ([more info](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)). Kindly add the `add_rental` method in your Book class :)